### DISCUSSION TITLE
preload warnings recently showing up

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#14511] [CLI - PWA mode] [answered]
In recent builds of my quasar (v2, vite) pwa, I have started to see this type of warning in my browser console:
```
The resource at “https://site.my.net/app/assets/quasar-lang-pack.23c03030.js” preloaded with link preload was not used within a few seconds. Make sure all attributes of the preload tag are set correctly.
```
And I am not quite sure what to do to suppress these, can anyone offer some advice?

**Comment by ssuess** [comment, marked as answer]
Does not one have any suggestions here? These are being added by the Quasar build process, not by me. Is there a way to remove these messages or perhaps tweak this config?

**Reply by pdanpdan** [reply]
See https://github.com/quasarframework/quasar/issues/12866

**Reply by ssuess** [reply]
Thanks for the link. Alas there has been no response to the issue other than from a bot.

**Reply by pdanpdan** [reply]
The solution is in the issue

**Reply by ssuess** [reply]
ah thanks, it seemed from reading it that the parameter was being ignored (and that true was assumed as default), but after setting it (and setting it to true) the messages do seem to have disappeared. :)

**Comment by ssuess** [comment]
Is this REALLY not a problem for anyone else?
